samples/security/psp/all-pods-psp.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
```

Comment:
Could you add in the first line like:
For details about using this yaml file, please refer to:
https://istio.io/docs/tasks/security/auth-sds/#increasing-security-with-pod-security-policies

Reply:
I pushed the change. PTAL